### PR TITLE
fix : Select component improvements

### PR DESCRIPTION
**Pull request by SteliosKon**
## Description

Added improvement regarding styles.

## Changes

— Improve styles of the scroll bar
— Improve spacing to match the dropdown spacing
— Multi-select, does not close when selecting an option
— Longer options do not wrap in the following line, a 3-dot suffix appears when word exceeds the width
— Add border-radius when hovering (only on select with inner search) — Select height now is correct (same as input)

### CODE REVIEW

File: src/components/FormElements/Select/Select.tsx [hunk 1]
```diff
@@ -62,11 +64,14 @@ const Select: ForwardRefRenderFunction<
   const [isFocused, setIsFocused] = useState(false);
   const [inputValue, setInputValue] = useState("");
 
+  // it is important to set the indicator size to the same height as the input to avoid centering issues
+  const indicatorSize = Number(inputHeight[size].replace("rem", "")) * 16 - 2;
```
Comment (ThrasyvoulosKafasis) [inline src/components/FormElements/Select/Select.tsx:68]: You can use `calc(${inputHeight[size]} - 0.125rem)` in indicatorsContainer height prop. Also add a comment that 0.125rem is border height for future refference.

File: src/components/FormElements/Select/styles.ts [hunk 2]
```diff
@@ -46,16 +42,28 @@ export const selectContainer = (
     }
 
     .control-${size} {
-      min-height: ${inputHeight[size]};
+      /* minus border */
+      min-height: calc(${inputHeight[size]} - 2px);
```
Comment (ThrasyvoulosKafasis) [inline src/components/FormElements/Select/styles.ts:46]: Use rem

**Review by ThrasyvoulosKafasis — CHANGES_REQUESTED** [review]

File: src/components/FormElements/Select/Select.tsx (hunk 1 above)
Comment (SteliosKon) [reply src/components/FormElements/Select/Select.tsx:68]: Unfortunately, calc cannot be used in js files.

File: src/components/FormElements/Select/styles.ts (hunk 2 above)
Comment (SteliosKon) [reply src/components/FormElements/Select/styles.ts:46]: I used px to clearly indicate that the border is removed

Comment (ThrasyvoulosKafasis) [reply src/components/FormElements/Select/styles.ts:46]: Leave as it is

File: src/components/FormElements/Select/Select.tsx (hunk 1 above)
Comment (ThrasyvoulosKafasis) [reply src/components/FormElements/Select/Select.tsx:68]: The styles var is a CSSObject, so the calculation it will work.

Comment (SteliosKon) [reply src/components/FormElements/Select/Select.tsx]: Worked [here](https://github.com/epignosis/gnosis/pull/199/commits/0a9c3a7659487179ca1f5446446da7160fc28412)

**Review by ThrasyvoulosKafasis — APPROVED** [review]

**Pull request merged**